Simplenote/Information/NoteInformationController.swift

```diff
@@ -142,13 +142,18 @@ private extension NoteInformationController {
         }
 
         let referenceRows = references.map { (note) -> Row in
+            let date = DateFormatter.dateFormatter.string(from: note.modificationDate)
+            let value = "\(Localization.interlinkReferences(references.count)), "
+                + "\(NSLocalizedString("Last Modified", comment: "Date of note last modified")) "
+                + date
```

Comment:
Lets extract `NSLocalizedString("Last Modified", comment: "Date of note last modified")` to `Localization` struct, so that all localization is in one place. Lets also add `date` as a parameter so that if needed the order of words can be changed during localization.

It can look like this: `NSLocalizedString("Last Modified %1$@", comment: "Date of note last modified. Parameter: %1$@ - formatted date")`